Discovery/DetectTorrentUse

```diff
@@ -0,0 +1,4 @@
+//Custom detection to find use of torrenting software or browsing related to torrents
+NetworkCommunicationEvents
```

Comment:
Please add a time filter, over last 1d or 7d, to make sure the query has good performance.
Take a look on other queries in this repo to see an examples.